sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala

```diff
@@ -302,6 +302,24 @@ case class CurrentUser() extends LeafExpression with Unevaluable {
   final override val nodePatterns: Seq[TreePattern] = Seq(CURRENT_LIKE)
 }
 
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = """_FUNC_() - connected user name.""",
+  examples = """
+    Examples:
+      > SELECT _FUNC_();
+       mockingjay
+  """,
+  since = "3.5.0",
+  group = "misc_funcs")
+// scalastyle:on line.size.limit
+case class SessionUser() extends LeafExpression with Unevaluable {
```

Comment:
why not register as an alias with CurrentUser

Reply:
Done, thank you!